app/services/calendar_service.rb

```diff
@@ -0,0 +1,75 @@
+class CalendarService
+  def create_google_client(user)
+    client = conn
```

Comment:
This method is a mouthful. Hopefully we will have time to refactor this before this project is due